### PR TITLE
Added changes from PR #10947. Bumped task version

### PR DESCRIPTION
**Pull request by anatolybolshakov**
**Task name**: XcodeV5

**Description**: reworked changes from https://github.com/microsoft/azure-pipelines-tasks/pull/10947 
- Bumped task version
- Added missed generated string resources

**Documentation changes required:** Y - created [ticket](https://github.com/microsoft/build-task-team/issues/180) for adding of documentation 

**Added unit tests:** N

**Attached related issue:** N

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/XcodeV5/xcode.ts [hunk 1]
```diff
@@ -239,6 +239,10 @@ async function run() {
             let xcPrettyPath: string = tl.which('xcpretty', true);
             let xcPrettyTool: ToolRunner = tl.tool(xcPrettyPath);
             xcPrettyTool.arg(['-r', 'junit', '--no-color']);
+            let xcPrettyArgs: string = tl.getInput('xcprettyArgs');
```
Comment (alexander-smolyakov) [inline Tasks/XcodeV5/xcode.ts:242]: nit: `const`

File: Tasks/XcodeV5/xcode.ts [hunk 2]
```diff
@@ -311,6 +315,11 @@ async function run() {
             if (useXcpretty) {
                 let xcPrettyTool: ToolRunner = tl.tool(tl.which('xcpretty', true));
                 xcPrettyTool.arg('--no-color');
+                let xcPrettyArgs: string = tl.getInput('xcprettyArgs');
```
Comment (alexander-smolyakov) [inline Tasks/XcodeV5/xcode.ts:318]: nit: `const`

File: Tasks/XcodeV5/xcode.ts [hunk 3]
```diff
@@ -460,6 +469,11 @@ async function run() {
                     if (useXcpretty) {
                         let xcPrettyTool: ToolRunner = tl.tool(tl.which('xcpretty', true));
                         xcPrettyTool.arg('--no-color');
+                        let xcPrettyArgs: string = tl.getInput('xcprettyArgs');
```
Comment (alexander-smolyakov) [inline Tasks/XcodeV5/xcode.ts:472]: nit: `const`

File: Tasks/XcodeV5/xcode.ts (hunk 1 above)
Comment (anatolybolshakov) [reply Tasks/XcodeV5/xcode.ts]: Thanks 👍  Fixed it

**Pull request merged**